app/assets/javascripts/bastion/components/nutupane.factory.js

```diff
@@ -69,6 +69,13 @@ angular.module('Bastion.components').factory('Nutupane',
                 params.search = table.searchTerm || "";
                 params.search = self.searchTransform(params.search);
 
+                table.resourceName = params.resourceName || "items"
+
+                table.emptySearchMessage = params.emptySearchMessage || 
+                                           "No " + table.resourceName + " returned for your search query.";
+                table.emptyRowsMessage = params.emptyRowsMessage ||
+                                         "You currently don't have any " + table.resourceName;
```

Comment:
These statements are untranslated.

Further more they belong in html and not in this JS file.